plugin/src/main/java/at/helpch/chatchat/util/ChannelUtils.java

```diff
@@ -33,4 +33,19 @@ private ChannelUtils() {
                 .filter(channel -> input.startsWith(channel.messagePrefix()))
                 .findFirst();
     }
+
+    public static @NotNull Optional<String> findChannelName(
+        @NotNull final Map<String, ChatChannel> channels,
+        @NotNull final ChatChannel channel
+    ) {
+        for (Map.Entry<String, ChatChannel> entry : channels.entrySet()){
```

Comment:
imagine not using streams /s

Reply:
Yes I know. but you said you're going to fix all that mess so I didn't even bother. Anyways I Fixed it in the latest commit I pushed a second ago.